### DISCUSSION TITLE
Organization vs community storage

### DISCUSSION THREAD

**Discussion by jeanbeeckman** [opengisch/QField#3925] [Q&A]
Hello,

I am planning to create an organization with four members. I would like the organization to have at least 2GB of storage capacity in order to accommodate a large project that may be larger than 1GB.

I am wondering if it is possible for all four members to have COMMUNITY (max 100MB storage capacity), or if they all need to have PRO accounts? In other words, is the storage capacity of each member inherited from the organization's total storage capacity?

I am not sure if it is possible to create a project within an organization in Qfieldsync, rather than as an individual user.

Thank you,
Jean

**Comment by ZsanettMed** [comment]
Hi Jean, 

First of all, your plan sounds good :) You can create an organization with 4 members with your personal user account, the organization will have its own billing account, which means a separate invoice. 

Your personal account and the accounts of the other 3 member can be either community or pro accounts, since any personal user can be a member of your organization. 

Organizations have 1 GB of storage included. Any additional storage has to be prepaid and has a fixed price of 5€ per month per GB. If you need 2GB in total, you have 1GB from your plan and 1GB of additional storage. The additional storage will cost you 5€.

The storage is the sum of the size of all files in all projects that are owned by the account (In your case the account of the organization). Therefore, storage capacity of each member is not inherited from the organization's total storage capacity.

Without creating an organization 4 members can not collaborate on the same project(s) only if the project is public. Public accounts can not have additional storage. Pro user can collaborate with one other pro user (not 3). The option -team management- is only available for organization accounts only. 

Here are some further information about organization accounts: 
To add a new organization member, you need to have their username or email first. Which means they need to register their own users.
When QFieldCloud charges the monthly bill for your organizational account, you pay only for the active organization members. By active, it's someone who either downloaded or uploaded to a organizational project. If the organization member did nothing in during the month, it is not considered active.
Note there is a minimal amount of 2 users that will be billed each month, no matter if there were active.

See also our [FAQs](https://qfield.cloud/faq.html)
Or other discussions like #3836 or #3832

If you upgrade till the 22nd of February you can get the EarlyBee 🐝 discount, which means 40% of the yearly price (instead 20€ - 12 € per user / month). The discount will automatically apply for the next 12 month. 

I hope I could help! Enjoy QFieldCloud :) 
Best, 
Zsanett

**Reply by jeanbeeckman** [reply]
Thanks for your reply. To clarify the part of storage issue : 

I remarked that even if I'm a member of an organization account, with **qfieldsync plugin**, I can only create project as member and not as organization.
So if we have an organisation with 4 community members the maximum size of each project will be 100Mb (because created as community account limited at 100 Mb) but we can store 20 projects before to reach the limit of our organisation (2Gb)
We often have project >100Mb because we also take photos on site.

Thanks for more explaination

Jean

**Comment by ZsanettMed** [comment]
Hello Jean, 
The storage capacity of the organization (1 GB) has to be considered separately from the user storage (100MB each). 
e.g. The 100MB is the own storage capacity of the user and has nothing to do with the organizations storage capacity. That means, you can have own projects (100MB) and organization projects (1GB). Other user do not have access to your own projects (100MB). You can sync projects to the organization where you are a member. As the owner of the organization you can also transfer projects from your user account to the organization in QFieldCloud and additionally you have the possibility to create in QFieldCloud projects in the organization where you are the owner.
<img width="307" alt="image" src="https://user-images.githubusercontent.com/83091224/220315508-c5409878-dc15-4e5b-b5eb-618306b35e2f.png">

**Reply by jkabubo** [reply]
what if alll data is stored in a seperate postgis remote post gis and you have 30 users  how use the project.

1) what space is uitised is this case.
3) What is the cost
